services/web-app/lib/github.js

```diff
@@ -806,6 +789,42 @@ const getLibraryAssets = async (params = {}) => {
   return assets
 }
 
+/**
+ * Validates and filters an array of assets,
+ * returns a modified array containing only valid assets for a library.
+ * @param {Promise<import('../typedefs').Library>} library
```

Comment:
Should this be a promise?
```suggestion
 * @param {import('../typedefs').Library} library
```

Reply:
true! thankss